build/status/aW5kX2Rpci9OTERBU18wLjlfMTM1LjEzNV8yMzEuMjM5X2Rsd3Jmc2ZjLm5jLmluZA.yml

```diff
@@ -0,0 +1,11 @@
+version: 0.3.0
+name: ind_dir/NLDAS_0.9_135.135_231.239_dlwrfsfc.nc.ind
```

Comment:
ahh, I like this suggestion, because there is going to be a ton of these .ind files, and like you say, I don't think they are relevant to other collaborators on this repo...

Reply:
ignoring 1_nldas_cubes/log/*.ind also ignores `1_nldas_append_rmk.ind` and `1_nldas_subset_rmk.ind` which I think I want to keep in source control...

I _think_ this works for me in .gitignore: `1_nldas_cubes/log/*.nc.ind`

Reply:
re: "These build/status files". Is it all or none w/ gitignoring these? I think I want to ignore all of the build.ymls related to the .nc files, but I don't think I want to ignore any of the other ones. Not sure that use-case is captured in https://github.com/USGS-R/scipiper/issues/48#issuecomment-395571189

Reply:
Oh, oops, I wasn't super clear in that comment - I've edited it now. Yes, I meant to selectively ignore just the build ymls related to the nc.ind files.